Firebase.cpp

```diff
@@ -74,21 +81,31 @@ String Firebase::makeURL(const String& path) {
   return url;
 }
 
-String Firebase::sendRequest(const char* method, const String& path, const String& value) {
-  _error.reset();
+int Firebase::sendRequest(const char* method, const String& path, const String& value) {
```

Comment:
maybe just return `void` here, since we already store the `statusCode` in the error object.